tutorials/define_and_compile_model.py

```diff
@@ -0,0 +1,72 @@
+"""
+Quick Start - Compile Resnet-18 Model for Raspberry Pi
+===========================================
+**Author**: `Yao Wang <https://github.com/kevinthesun>`_
+
+This example shows how to build a neural network with NNVM python frontend and generate runtime library for Raspberry Pi with TVM.
```

Comment:
We may want to list the currently-supported hardware targets. @tqchen Please point out that where we can find this kind of information. Perhaps `tvm.target`?

Reply:
we did not yet support target listing, but we do have things like tvm.codegen.enabled and tvm.module.enabled which can query whether a target is available

Reply:
https://github.com/dmlc/tvm/blob/master/python/tvm/target.py#L70
Maybe we can list major target names here?

Reply:
we can reuse pics like http://tvmlang.org/2017/10/30/Bringing-AMDGPUs-to-TVM-Stack-and-NNVM-Compiler-with-ROCm.html the last part